CONTRIBUTING.md

```diff
@@ -300,13 +300,18 @@ def example_function(argument: argument_type) -> return_type:
 
     Parameters
     ----------
-        argument: argument_type
-            Description of your argument.
+    argument: argument_type
```

Comment:
This is what I'm talking about for the parameter indentation, so just disregard all of this :) Glad we're using the correct formatting 😊